app/src/main/java/com/xmartlabs/daydreaming/model/Video.java

```diff
@@ -0,0 +1,24 @@
+package com.xmartlabs.daydreaming.model;
+
+import org.parceler.Parcel;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Created by chaca on 4/21/17.
+ */
+@Builder
+@Data
+public class Video {
+  String id;
+  String theme;
+  String type;
+
+  public String getId() {
```

Comment:
Non `@Data` neither `@Getter` were working well, so I didn't have another option to do it like this. When I say working well I mean that it didn't generate the get method for the attributes of the class.